### ISSUE TITLE
Compile for Apple Vision

### ISSUE THREAD

**Issue by kjwamlex** [stasel/WebRTC#76]
I don't think it should be far away from building one for Apple Vision as it should be similar to iOS. I am not asking for Apple Vision running as iPad app, I am asking for Apple Vision itself and simulator. Please compile a framework for Apple Vision please.

**Comment by ethan021021** [comment]
This would be a great addition

**Comment by ryanbliss** [comment]
+1. Any update here would be greatly appreciated. 

P.S. I found this thread, not sure how relevant it is: https://groups.google.com/g/discuss-webrtc/c/pdJrg24joGA?pli=1

**Comment by TR-MZ** [comment]
Agree! I have to use a webkit view now for my app :/ not very optimized . Please we need vision os port

**Comment by devluz** [comment]
I have created a port here: https://github.com/because-why-not/webrtc_visionpro_workspace
It would be great to get more feedback by someone with an actual device that can test it in their own project.
